_posts/2022-03-25-conformation-generation.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: post
+title: Generating Molecular Conformations via Normalizing Flows and Neural ODEs
+authors: Mukundh Murthy, Nikhil Devraj
+tags: [cheminformatics, drug discovery, generative modeling]  # This should be the relevant areas related to your blog post
+---
+In this post, we provide an in-depth overview of methods outlined in the paper "Learning Neural Generative Dynamics for Molecular Conformation Generation," discuss the impact of the work in the context of other conformation generation approaches, and additionally discuss future potential applications to improve the diversity and stability of generated conformations.
+
+1. [Introduction](#introduction)
+2. [An Overview of the Deep Generative Approach](#generative-overview)
+3. [Modeling distributions of distances](#distances)
+4. [Modeling distributions of conformations](#conformations)
+5. [Sampling](#sampling)
+6. [Future Work](#future-work)
+7. [References](#references)
+
+## Introduction
+
+In drug discovery, generating molecular conformations is useful across a variety of applications. For example, docking of various molecular 3D conformations to a specific protein allows drug hunters to decide whether a small molecule binds to a specific pocket in a multitude of conformations or a select few.
+
+<p align="center">
+  <img src="{{ site.url }}/public/images/2022-03-25-conformation-generation/vina.jpg" style="width: 300px">
+style="width: 300px">
+  <b>Figure 1:</b> Autodock Vina is a computer program that takes a given 3D conformation of a molecule and protein and predicts the binding free energy. An algorithm like the one discussed in this blog could generate a wide variety of conformations for Autodock Vina to test. ([Source](https://vina.scripps.edu/))
```

Comment:
note that I don't think markdown shortcuts for hyperlink work in an html block. This whole block could be updated to:

```
<p align="center">
  <img src="{{ site.url }}/public/images/2022-03-25-conformation-generation/vina.jpg" style="width: 300px">
  <b>Figure 1:</b> Autodock Vina is a computer program that takes a given 3D conformation of a molecule and protein and predicts the binding free energy. An algorithm like the one discussed in this blog could generate a wide variety of conformations for Autodock Vina to test. (<a href="https://vina.scripps.edu/">Source</a>)
</p>
```